src/grammar/query.rs

```diff
@@ -118,7 +118,7 @@ fn parse_column_expr(p: &mut Parser) {
         p.start(SyntaxKind::ColumnExpr);
 
         parse_expr(p);
-        if p.at(T![as]) || p.at(T![quoted_ident]) {
+        if p.at(T![as]) || p.at(T![quoted_ident]) || p.at(T![unquoted_ident]) {
```

Comment:
In other locations we use constructs such as `[T![elsif], T![else], T![end]].contains(&p.current())`, up to you what you want to use here.